charts/kafdrop/values.yaml

```diff
@@ -0,0 +1,1034 @@
+nameOverride: ""
+
+## Overwrite Fullname Template
+# fullnameOverride -- Overwrite `lib.utils.common.fullname` output
+fullnameOverride: ""
+
+## Common Labels
+# commonLabels -- Common Labels are added to each kubernetes resource manifest.
+commonLabels: {}
+
+## Overwrite Labels
+# overwriteLabels -- Overwrites default labels, but not resource specific labels and common labels
+overwriteLabels: {}
+
+## Selector Labels
+# selectorLabels -- Define default [selectorLabels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+selectorLabels: {}
+
+## Version Capabilities
+# kubeCapabilities -- Overwrite the Kube GitVersion
+# @default -- `$.Capabilities.KubeVersion.GitVersion`
+kubeCapabilities: ""
+
+## Proxy Values
+##
+proxy:
+
+  ## HTTP Proxy Configuration
+  httpProxy:
+
+    ## HTTP Proxy Host Configuration
+    # proxy.httpProxy.host -- Configure HTTP Proxy Hostname/IP (without protocol://)
+    host: ""
+
+    ## HTTP Proxy Port Configuration
+    # proxy.httpProxy.port -- (int) Configure HTTP Proxy Port
+    port:
+
+    ## HTTP Proxy Protocol Configuration
+    # proxy.httpProxy.protocol -- Configure HTTP Proxy Protocol (http/https)
+    # @default -- http
+    protocol: ""
+
+  ## HTTPS Proxy Configuration
+  httpsProxy:
+
+    ## HTTPS Proxy Host Configuration
+    # proxy.httpsProxy.host -- Configure HTTPS Proxy Hostname/IP (without protocol://)
+    host: ""
+
+    ## HTTP Proxy Port Configuration
+    # proxy.httpsProxy.port -- (int) Configure HTTPS Proxy Port
+    port:
+
+    ## HTTP Proxy Protocol Configuration
+    # proxy.httpsProxy.protocol -- Configure HTTPS Proxy Protocol (http/https)
+    # @default -- http
+    protocol: ""
+
+  ## No Proxy Hosts Configuration
+  # proxy.noProxy -- Configure No Proxy Hosts
+  # noProxy: [ "localhost", "127.0.0.1" ]
+  noProxy: []
+
+## Define Container Timezone
+# timezone -- Define Container Timezone (Sets TZ Environment)
+timezone: "Europe/Zurich"
+
+
+## Kafdrop Configuration
+## Defines values which will be set as environment variable for the
+## kafdrop pod.
+##
+config:
+
+  ## Kafka SubElement
+  ##
+  kafka:
+
+    ## Define Connection Endpoints to Kafka
+    ## brokers. Can be of type array/slice or
+    ## a comma separated string
+    ##
+    connections:
+      - localhost:9092
+
+    ## Kafka Properties Configuration
+    ##
+    properties:
+      ## Optional Content for Kafka Properties
+      content: |
+        {{ toString "sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='suchUser' password='veryS3CRET';\nsecurity.protocol=SASL_PLAINTEXT\nsasl.mechanism=PLAIN" | b64enc }}
+      destination: "kafka.properties"
+
+    # Truststore Configuration
+    truststore:
+      ## Optional Content for Kafka Truststore
+      content: ""
+      destination: "kafka.truststore.jks"
+
+    ## Keystore Configuration
+    ##
+    keystore:
+      ## Optional Content for Keystore
```

Comment:
Please describe in the comment, that the value of "content" should be a base64 encoded jks file.

Reply:
implemented.